<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ifcManagementWindow</class>
 <widget class="QMainWindow" name="ifcManagementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage IFC Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="list_IFC_management">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>491</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BIMModels_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BIM Models</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>77</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_import_ifc">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete_ifc">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
